app/uk/gov/hmrc/trackingconsentfrontend/assets/src/interfaces/isFeatureEnabled.ts

```diff
@@ -0,0 +1,22 @@
+import Cookies from 'js-cookie'
+
+const isFeatureEnabled = (featureName: String) => {
+    function getEnabledFlagFromCookie(featureName: String) {
+        return (Cookies.get(featureName) == 'true')
+    }
+
+    function getEnabledFlagFromQuery(featureName: String) {
+        return window.location.search.includes(featureName.concat('=true'))
```

Comment:
Would an ES6 template literal be easier to read here vs .concat?

Reply:
Done.